backend/mora/service/employee.py

```diff
@@ -483,14 +483,33 @@ def terminate_employee(employee_uuid):
         )
     ]
 
+    trigger_dict = {
+        'role_type': mapping.EMPLOYEE,
+        'event_type': Trigger.Event.ON_BEFORE,
+        'request': request,
+        'request_type': handlers.RequestType.TERMINATE,
+        'uuid': employee_uuid,
+        'employee_uuid': employee_uuid
+    }
+
+    if not flask.request.args.get('triggerless'):
+        Trigger.run(trigger_dict)
+
     for handler in request_handlers:
         handler.submit()
 
+    trigger_dict["event_type"] = Trigger.Event.ON_AFTER
+    trigger_dict["result"] = result = flask.jsonify(employee_uuid)
+
+    if not flask.request.args.get('triggerless'):
```

Comment:
We have a utility function that allows you to specify this as a boolean, interpreting `0`, 'no' and 'false' as expected:
```suggestion
    if not util.get_args_flag('triggerless'):
```

Reply:
rettet ind